app/src/main/res/values-ca/strings.xml

```diff
@@ -217,7 +217,7 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">An offline gallery for managing your files without ads, respecting your privacy.</string>
+    <string name="app_short_description">Una galeria sense connexió per gestionar els vostres fitxers sense anuncis, respectant la vostra privadesa.</string>
```

Comment:
please make this shorter than 80 characters, just as mentioned on the line above :) Ill still accept the Pull request, correcting it with a new one will be easier than modifying this one. Thanks